docs/releasing-files.md

```diff
@@ -1,115 +1,191 @@
+ 
+The solution for this is to use rounding across the entire report (to the nearest 5, 7, or 10). Rounding eliminates the need for output checkers to compare different versions of the report outputs, as no small increases can occur. It is not likely that rounding will significantly affect the report in any meaningful way. Statistical analyses can be carried out prior to rounding and the results displayed as normal provided they are classed as low risk. If only charts are shown, rounding is not always necessary, but charts should have sufficiently low resolution that any small number changes cannot be precisely determined. However, numbers could be rounded nonetheless, as this would not noticeably affect the figures. 
 
-Where possible it should be clear what has been redacted, so for example do not redact table titles and category names. 
-By convention redactions take the form `[REDACTED]` to make redacted elements easier to search for.
-	
-Ideally there shouldn't be any files in the folder that are not intended for release (whether potentially disclosive or not) as these should have been removed by the analysis scripts or placed elsewhere. 
-If they are there, delete them.
+**Running similar studies**
+There may be cases where you have run an analysis and results have been released following approval from the output checking team, but at a later date you decide you want to make changes to the analysis such as adding an extra code to a codelist. This may result in small changes in the outputs that can be disclosive, ie, <=5 individuals have the new code recorded. In these cases, you may need to wait until there has been enough change in the underlying study population (due to the movement of patients between practices) before rerunning the study.
+ 
+If you are likely to release data multiple times, e.g. for initial discussion with collaborators, use rounding of outputs initially and/or a threshold substantially higher than 5 for suppressing low numbers. 
 
-This current approach to disclosure control is conservative and deliberately reduces the need for judgement calls. 
-It may be possible for exceptions to be made if they can be justified as being both analytically necessary and definitely non-disclosive. This must be discussed with the OpenSAFELY team.
+## 2. Requesting release of outputs from the server
 
-**If you are unsure about anything, please email us: <disclosurecontrol@opensafely.org>.**
+Having applied disclosure controls to the aggregated study data, you are ready to request their release. **Only specific members of the OpenSAFELY team trained in output checking have permissions to release the data**. When you are ready to request a release of your aggregated results, and only the minimum outputs necessary please [complete this form](mailto:datarelease@opensafely.org) and email us at **datarelease@opensafely.org**. For each output wishing to be released you will need to provide a clear contextual description including:
```

Comment:
link to form not correct - linked to email by mistake 

also email link needs the <email> added